src/graphql/queries/roles.graphql

```diff
@@ -0,0 +1,7 @@
+query GetColonyRoles($filter: ModelColonyRoleFilterInput) {
+  listColonyRoles(filter: $filter) {
```

Comment:
Could you potentially use `getRoleByDomainAndColony` instead? It's recommended by AWS: https://docs.amplify.aws/gen1/javascript/build-a-backend/graphqlapi/data-modeling/#configure-a-secondary-index